sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -58,25 +59,18 @@
         </v-col>
       </v-row>
       <v-row>
-        <!-- Address 1 -->
+        <!-- Address -->
         <v-col>
-          <title-value propertyTitle="Address line 1" />
+          <title-value propertyTitle="Address" />
           <span
             class="label-value muted-content clearfix"
-            v-for="addressLine in addressList1(item)"
+            v-for="addressLine in addressList(item)"
             :key="addressLine"
             data-cy="institutionAddress1"
           >
             {{ addressLine }}
           </span>
         </v-col>
-
-        <!-- Address 2 -->
-        <v-col>
```

Comment:
for my info, so, there won't be address 2 ?

Reply:
Not as a separate column. It was supposed to be the "Address Line 2" (if I am not wrong).

Reply:
mm, actually I think, the old plan was that there can be 2 addresses for the institution (although we don't have anything in place to capture the 2nd address now), `Address 1` and `Address 2`

Reply:
Yes, maybe that also. There was a plan for a sort of `home address`/`mailing address` but it was dropped a long time ago.
Either way, seems to be pretty simple now.